macOS/DuckDuckGo/Favicons/Model/FaviconImageCache.swift

```diff
@@ -22,52 +22,39 @@ import Common
 import BrowserServicesKit
 import os.log
 
-@MainActor
 protocol FaviconImageCaching {
 
     init(faviconStoring: FaviconStoring)
 
+    @MainActor
     var loaded: Bool { get }
 
+    @MainActor
     func load() async throws
```

Comment:
it seems we don‘t need the `load()` method to be fully `@MainActor` covered: `FaviconStore.loadFavicons()` might be run in the background (while `loaded` and `entries` should be covered with `@MainActor` on the class level in this architecture)

Reply:
I’ve updated this function to not require main actor, and only populate `entries` on the main actor 🫡